src/components/gamification/BadgeStudio.tsx

```
@@ -0,0 +1,167 @@
+
+    const updateComposition = (changes: Partial<BadgeComposition>) => {
+        setComposition({ ...composition, ...changes });
+    };
+
+    const handlePublish = () => {
+        if (!title.trim()) {
+            toast.error("Please give the badge a title.");
+            return;
+        }
+
+        createBadge.mutate(
```

ESLint

[error] Replace `········` with `····`

(prettier/prettier)